documentation/src/main/markdown/gettingstarted/terms/detectors.md

```diff
@@ -1,5 +1,11 @@
 # Detectors
 
-[solution_name] uses detectors to find and extract dependencies from all supported package managers. For example, the Maven detector, which is run by default, executes an mvn dependency:tree command against a Maven project and derives dependency information, which can be sent to [blackduck_product_name]
+The [solution_name] Detector tool runs one or more detectors to find and extract dependencies from all supported package managers.
+
+Each package manager ecosystem is assigned a detector type. Each detector type may have multiple methods (detectors) used to extract dependencies.
```

Comment:
Each detector type may use multiple methods (detectors) to extract dependencies.

Reply:
I will add that. Thank you.